packages/design-system/src/components/FilterChip/FilterChip.tsx

```diff
@@ -56,25 +60,29 @@ export class FilterChip extends React.Component<FilterChipProps> {
 
   render(): React.ReactNode {
     const { id, label, ariaClearLabel, className, useAlternateIcon, size } = this.props;
+
     const buttonClassNames = classNames(
       'ds-c-filter-chip__button',
       size && size === 'big' ? 'ds-c-filter-chip__button--big' : '',
       className
     );
+
     const iconContainerClassNames = classNames(
       'ds-c-filter-chip__clear-icon-container',
       useAlternateIcon ? 'ds-c-filter-chip__clear-icon-alternate-container' : ''
     );
+
     return (
       <>
         <button
           className={buttonClassNames}
           id={id || uniqueId(`filter_`)}
           onClick={this.handleClick}
           onKeyDown={this.handleKeyDown}
+          aria-label={`. ${ariaClearLabel} ${label}`}
```

Comment:
[question] is the `.` intentional here?

Reply:
Yup! That was my suggestion. The period is intended to provide a pause for a screen reader so the distinction between the filter chip's name and the control to close it are clearer, so something like:

`New York` *pause* `Remove filter` instead of `New Yorkremove filter`.

Reply:
When I use VoiceOver and Safari and focus on one of these buttons, I hear only the aria-label. What you said above, Eric, makes me think that it should announce `Example big filter chip. Remove Small Example big filter chip`, but what I hear is `. Remove Small Example big filter chip, button`. Are you and Sarah getting different results?

Reply:
I'm not entirely sure what the AX should be for the readout. I tried following Target's example, knowing they invest pretty heavily in a11y, and their chips read out: "Remove filter [filter name ]". So Patrick's readout is the intended way.

Reply:
It doesn't really make sense, though, because don't we want that pause to be here instead `Remove Small . Example big filter chip, button` ?

Reply:
But the period is at the beginning and there's no break between the `ariaClearLabel ` and the `label` content

Reply:
I'd put the `.` break there at @ericwbailey direction. I understood this as providing a break between the previous filter chip read and the current one. I might be wrong about that though.

Reply:
I'm just telling ya'll what I hear and see in VoiceOver. Do you get a different result with VoiceOver on your machine?